dev/jobs/awips.sh

```diff
@@ -0,0 +1,25 @@
+#! /usr/bin/env bash
+
+set -x
+source "${HOMEgfs}/ush/load_fv3gfs_modules.sh"
+status=$?
+if (( status != 0 )); then exit "${status}"; fi
```

Comment:
Please use `[[ ]]` and make if blocks multiple lines.  `(( ))` logical tests do not log as well as `[[ ]]` and multiline if blocks are easier to read.

```suggestion
if [[ ${status} -ne 0 ]]; then
    exit "${status}"
fi